test/unittests/iotools/converters/adni_to_bids/test_adni_utils.py

```diff
@@ -27,6 +27,72 @@ def test_bids_id_to_loni(input_value, expected):
     assert bids_id_to_loni(input_value) == expected
 
 
+def test_adni_study_error():
+    from clinica.iotools.converters.adni_to_bids.adni_utils import ADNIStudy
+
+    with pytest.raises(
+        ValueError,
+        match="Invalid study name for ADNI: foo.",
+    ):
+        ADNIStudy.from_string("foo")
+
+
+@pytest.mark.parametrize(
+    "study_name,visit_code,expected",
+    [
+        ("ADNI3", "bl", "ADNI Screening"),
+        ("ADNI3", "M000", "ADNI3 Year 0.0 Visit"),
+        ("ADNI3", "M006", "ADNI3 Year 0.5 Visit"),
+        ("ADNI3", "M012", "ADNI3 Year 1.0 Visit"),
+        ("ADNI3", "M013", "ADNI3 Year 1.0833333333333333 Visit"),
```

Comment:
For example, this is a weird behavior. I would expect integer values for years...
I suspect the casting to integer wasn't correctly done in the first place (might be a parenthesis order issue??).